### PR TITLE
Ensure data sources are not loaded before saved preferences have been loaded

### PR DESCRIPTION
**Pull request by TonioF**
Hopefully solves #56. Unfortunately, as I could not reproduce the problem with the "Loading data sources ..." panel locally, I cannot tell for sure before we try this on our development cluster.

### CODE REVIEW

File: src/renderer/actions.ts [hunk 1]
```diff
@@ -490,6 +489,7 @@ export function loadPreferences(): ThunkAction {
             dispatch(loadInitialWorkspace(
                 getState().session.reopenLastWorkspace,
                 getState().session.lastWorkspacePath));
+            dispatch(loadDataStores());
```
Comment (forman) [inline src/renderer/actions.ts:492]: Loading data stores after loading workspace doesn't seem right. That way, opening datasets from last opened workspace will not be possible.

**Review by forman — CHANGES_REQUESTED** [review]
Check, order of resoureces loaded. Inital workspace + saved workflow must be able to resolve dataset references.

File: src/renderer/actions.ts (hunk 1 above)
Comment (forman) [reply src/renderer/actions.ts:492]: I suggest finding the root cause (you might have come close) then solve the root problem.

Comment (TonioF) [reply src/renderer/actions.ts]: The problem occurs here: [dispatch(loadDataStores());](https://github.com/CCI-Tools/cate-app/blob/85e155aceda00de09de077f4544e53d7c4f38382/src/renderer/actions.ts#L971)

If the previous session has not been loaded yet, the session value of selectedDataStoreId is null and set to the id of the first list entry - which is why you will not encounter this issue for the first data store (cci-store). The solution is to load in the data stores after the session has been loaded.

**Comment by AliceBalfanz** [comment]
Tested locally - and also could reproduce problem locally. Works well :) @forman requested I merge directly :)

**Pull request merged**